src/lib/common/statistics.h

```diff
@@ -315,9 +316,9 @@ extern int noOfSimulatedNotifications;
 
 /* ****************************************************************************
 *
-* timingStatistics - 
+* renderTimingStatistics -
 */
-extern std::string timingStatistics(std::string indent, Format format, std::string apiVersion);
+extern std::string renderTimingStatistics(void);
```

Comment:
In my opinion, the "user saving" (avoid a `| python -mjson.tool` or `| jsonlint` or similar at the end of the curl command) doesn't worth a more complex implementation at CB code.